README.md

```diff
@@ -61,6 +61,7 @@ Various CMake options are available to customize the client/server SDK builds.
 | `LD_BUILD_CONTRACT_TESTS` | Whether the contract test service (used in CI) is built.                                                                                          | Off                                                    | `BUILD_TESTING`                           |
 | `LD_BUILD_EXAMPLES`       | Whether example apps (hello world) are built.                                                                                                     | On                                                     | N/A                                       |
 | `LD_BUILD_SHARED_LIBS`    | Whether the SDKs are built as static or shared libraries.                                                                                         | Off  (static lib)                                      | N/A                                       |
+| `LD_BUILD_EXPORT_SYMBOLS` | Whether export C++ symbols.                                                                                                                       | Off  (hidden)                                          | `LD_BUILD_SHARED_LIBS`                    |
```

Comment:
Hello @puncha,

First, thank you for the contribution.

The nature of this feature mean it is likely to cause crashes, memory safety issues, and other undefined behaviors. I think it is reasonable to provide this for those who understand the implications, but we will need some additional warning language.

Maybe something like `Whether to export C++ symbols in shared libraries. C++ does not have an ABI, so only use this feature if you understand the risk and requirements. A mismatch in ABI could cause crashes or other unexpected behaviors.`

Reply:
Thank you for the review. I'll update my change soon.

Reply:
@kinyoklion Sorry for the late update. Thank you for the good suggestion! I've updated the description of the cmake option. Please review again.